src/assets/tocIcon.svg

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="17px" height="15px" viewBox="0 0 17 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>1B613A8A-961A-4ECD-B481-F2E0D68138BB</title>
```

Comment:
I think this is either not required or should be something readable

Reply:
same as above